apps/vaporgui/VSpinBox.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <string>
+
+#include <QWidget>
+#include <QSpinBox>
+
+#include "VContainer.h"
+
+//! class VSpinBox
+//!
+//! Wraps a QSpinBox and provides vaporgui's standard setter/getter functions
+//! and signals.
+
+class VSpinBox : public VContainer {
+    Q_OBJECT
+
+public:
+    VSpinBox( int min, int max);
```

Comment:
Maybe we rename it to `VIntSpinBox` to be better distinguish from `VDoubleSpinBox`?

Reply:
Done